### PR TITLE
Deprecate fastcgi

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
Following FastCGI deprecation https://github.com/openmicroscopy/openmicroscopy/pull/4056 doc should also be updated

### CODE REVIEW

File: omero/sysadmins/unix/install-web/install-wsgi.txt [hunk 1]
```diff
@@ -89,6 +89,18 @@ Start the Gunicorn worker processes running one thread listening on 127.0.0.1:40
     ... static files copied to '/usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static'.
     Starting OMERO.web... [OK]
 
+.. note::
+    Additional setting can be configured from command line arguments:
+
+    * --workers WORKERS - The number of worker processes for handling requests.
+
+    * --worker-connections WORKER_CONNECTIONS - he maximum number of
```
Comment (mtbc) [inline omero/sysadmins/unix/install-web/install-wsgi.txt]: the

File: omero/sysadmins/unix/install-web/install-fastcgi.txt [hunk 2]
```diff
@@ -1,5 +1,11 @@
-OMERO.web FastCGI deployment
-============================
+OMERO.web FastCGI deployment (DEPRECATED)
+=========================================
+
+.. note::
+    Deprecated since version 5.1.4: OMERO.web deployment using FastCGI
+    is deprecated and will be removed in OMERO 5.2.
+    Use (see :doc:`install-wsgi`)
```
Comment (hflynn) [inline omero/sysadmins/unix/install-web/install-fastcgi.txt]: This last line reads rather strangely, I'd remove the brackets and 'see' and make it:
Use `:doc:`install-wsgi` as an alternative.`

File: omero/sysadmins/unix/install-web/install-wsgi.txt [hunk 3]
```diff
@@ -89,6 +89,18 @@ Start the Gunicorn worker processes running one thread listening on 127.0.0.1:40
     ... static files copied to '/usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static'.
     Starting OMERO.web... [OK]
 
+.. note::
+    Additional setting can be configured from command line arguments:
```
Comment (hflynn) [inline omero/sysadmins/unix/install-web/install-wsgi.txt]: setting**s** plural

File: omero/sysadmins/unix/install-web/install-wsgi.txt [hunk 4]
```diff
@@ -89,6 +89,18 @@ Start the Gunicorn worker processes running one thread listening on 127.0.0.1:40
...
+
+    * --workers WORKERS - The number of worker processes for handling requests.
```
Comment (hflynn) [inline omero/sysadmins/unix/install-web/install-wsgi.txt]: The -> the

File: omero/sysadmins/unix/install-web/install-wsgi.txt [hunk 5]
```diff
@@ -89,6 +89,18 @@ Start the Gunicorn worker processes running one thread listening on 127.0.0.1:40
...
+
+    * --worker-connections WORKER_CONNECTIONS - he maximum number of
+      simultaneous clients.
+
+    * --wsgi-args WSGI_ARGS - Additional arguments. For more details
```
Comment (hflynn) [inline omero/sysadmins/unix/install-web/install-wsgi.txt]: Additional -> additional

**Comment by hflynn** [comment]
https://www.openmicroscopy.org/site/support/omero5.1-staging/sysadmins/unix/install-web.html needs some more changes, might be easiest for me to do them, I can push a commit to this PR tomorrow.

File: omero/sysadmins/unix/install-web/install-wsgi.txt [hunk 6]
```diff
@@ -89,6 +89,18 @@ Start the Gunicorn worker processes running one thread listening on 127.0.0.1:40
     ... static files copied to '/usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static'.
     Starting OMERO.web... [OK]
 
+.. note::
```
Comment (hflynn) [inline omero/sysadmins/unix/install-web/install-wsgi.txt]: Does this need to be in a note box?

File: omero/sysadmins/unix/install-web/install-wsgi.txt (hunk 4 above)
Comment (hflynn) [inline omero/sysadmins/unix/install-web/install-wsgi.txt]: Also, the double `--` are not formatting on staging, this needs to be marked up with back ticks

File: omero/sysadmins/unix/install-web/install-wsgi.txt [hunk 7]
```diff
@@ -89,6 +89,17 @@ Start the Gunicorn worker processes running one thread listening on 127.0.0.1:40
     ... static files copied to '/usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static'.
     Starting OMERO.web... [OK]
 
+Additional settings can be configured from command line arguments:
+
+* `--workers WORKERS` - the number of worker processes for handling requests.
+
+* `--worker-connections` WORKER_CONNECTIONS - the maximum number of
```
Comment (hflynn) [inline omero/sysadmins/unix/install-web/install-wsgi.txt]: Should the WORKER_CONNECTIONS bit also be inside the mark-up here?

**Comment by hflynn** [comment]
@aleksandra-tarkowska one question above and then I'll fix up the rest of this on a new PR so you don't have to bother with it anymore.

**Comment by aleksandra-tarkowska** [comment]
try now :)

File: omero/sysadmins/unix/install-web/install-wsgi.txt [hunk 8]
```diff
@@ -89,6 +89,17 @@ Start the Gunicorn worker processes running one thread listening on 127.0.0.1:40
     ... static files copied to '/usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static'.
     Starting OMERO.web... [OK]
 
+Additional settings can be configured from command line arguments:
```
Comment (aleksandra-tarkowska) [inline omero/sysadmins/unix/install-web/install-wsgi.txt]: @hflynn this is an output of web start -h if we have any better way to format it go for it.

**Comment by hflynn** [comment]
Thanks @aleksandra-tarkowska I'm going to fetch your branch and open a new PR to finish this up.

**Pull request closed without merging**